conf/article_validator.yaml

```diff
@@ -0,0 +1,721 @@
+          - 001 #white
+          - 003 #off-white
+          - 010 #transparent
+          - 958 #coloured denim
+          - 505 #turquoise
+          - 515 #mottled turquoise
+          - 104 #silver
+          - 801 #anthracite
+          - 802 #black
+          - 811 #mottled anthracite
+          - 812 #mottled black
+          - 859 #metallic black
+          - 301 #light red
+          - 302 #red
+          - 303 #dark red
+          - 304 #bordeaux
+          - 305 #coral
+          - 311 #mottled light
+          - 312 #mottled red
+          - 313 #mottled dark
+          - 314 #mottled bordeaux
+          - 706 #copper
+          - 315 #metallic red
+          - 401 #rose
+          - 402 #pink
+          - 411 #mottled rose
+          - 412 #mottled pink
+          - 915 #neon pink
+          - 916 #nude
+          - 605 #petrol
+          - 615 #mottled teal
+          - 204 #apricot
+          - 611 #mottled light
+          - 612 #mottled green
+          - 613 #mottled dark
+          - 913 #neon green
+          - 616 #metallic green
+          - 101 #light grey
+          - 102 #grey
+          - 103 #dark grey
+          - 105 #stone
+          - 111 #mottled light
+          - 112 #mottled grey
+          - 113 #mottled dark
+          - 956 #grey denim
+          - 114 #metallic grey
+          - 115 #gunmetal
+          - 206 #gold
+          - 217 #rose gold
+          - 201 #light yellow
+          - 202 #yellow
+          - 203 #dark yellow
+          - 211 #mottled light
+          - 212 #mottled yellow
+          - 213 #mottled dark
+      supplierColor:
+        description: |
+          This is the color that was provided by the articles manufacturer or
+          supplier.
+        type: string
+        minLength: 3
+        maxLength: 50
+      supplierSKU:
+        description: |
+          Supplier SKU of ArticleConfig valid for specific color scheme of
+          Article described by ArticleConfig
+        type: string
+      genders:
+        description: |
+          Genders for which to
+          Product is sold. Multiple Genders
+          are possible
+        type: array
+        items:
+          type: string
+          enum:
+            - female
+            - male
+        maxItems: 2
+      season:
+        description: |
+          Available
+          Seasons
+          are: HW[XX] -
+          "Herbst-Winter"
+          (Fall/Winter)
+          where as XX
+          represents the year as two
+          digit
+          number, FS[XX] -
+          "Frühjahr-Sommer"
+          (Spring/Summer) where as XX
+          represents the year as two
+          digit
+          number, NOS (Never
+          Out
+          of Stock) - can always be
+          sold
+        type: array
+        items:
+          type: string
+          minLength: 3
+          maxLength: 4
+          pattern: /((HW|FS)[0-9]{2,2}|NOS|HWBASIS|FSBASIS|GANZJAHR|AUTUMN_WINTER_BASICS|SPRING_SUMMER_BASICS|YEAR_ROUND_BASICS)/
```

Comment:
Would be great if we can have a consistent system here, for example:
[
   [Season, Year],
   [Season, Year],
   [Season, Year],
   [Season, Year]
]

Where season in one of four and year is year :)

Validate if special cases can be dropped in favour of consistent normalized format or are they legacy.